searches/exponential_search

```diff
@@ -0,0 +1,78 @@
+
+<?php 
+// PHP program to find an element x in a 
+// sorted array using Exponential search. 
+  
+// Returns position of first  
+// ocurrence of x in array 
+function exponentialSearch($arr, $n, $x) 
+{ 
+      
+    // If x is present at  
+    // first location itself 
+    if ($arr[0] == $x) 
+        return 0; 
+  
+    // Find range for binary search 
+    // by repeated doubling 
+    $i = 1; 
+    while ($i< $n and $arr[$i] <=$x) 
+        $i = $i * 2; 
+  
+    // Call binary search  
+    // for the found range. 
+    return binarySearch($arr, $i / 2,  
+                        min($i, $n), $x); 
+} 
+  
+// A recursive binary search 
+// function. It returns location 
+// of x in given array arr[l..r] is 
+// present, otherwise -1 
+function binarySearch($arr, $l,  
```

Comment:
You will likely need to change this function's name or change this file to a class since we already have a `binarySearch` function in the `Searches` file directory.